### DISCUSSION TITLE
Deno support in README is gone.

### DISCUSSION THREAD

**Discussion by c608345** [faker-js/faker#1329] [Q&A]
Does faker js run in Deno ?

**Comment by Shinigami92** [comment]
We have a section for how it could work in Deno here: https://fakerjs.dev/guide/#cdn-deno
Maybe if you encounter an issue it could be that it is not based on faker itself but Deno :thinking: https://twitter.com/DasSurma/status/1565664306334777344

**Comment by c608345** [comment]
I think deno should put here with node https://fakerjs.dev/guide/#environments.

**Reply by Shinigami92** [reply]
Please also provide >**why**< you would put it there

**Reply by c608345** [reply]
There are bun, deno , node.js.   With clear description, user will knonw where the library can run.

**Reply by ST-DDT** [reply]
@Shinigami92 Do we officially support deno? Is there anything to support? Because we don't have a build pipeline to check that.

**Reply by Shinigami92** [reply]
> @Shinigami92 Do we officially support deno? Is there anything to support? Because we don't have a build pipeline to check that.

I don't have more knowledge as you have.
But as @c608345 mentioned, there are coming some new "engines" like bun.
I assume it's a bit like Java vs Kotlin and Scala. Kotlin is small in terms of userbase but IMO it's there task to be compatible with node.
I might be wrong with that assumption.
